homeassistant/components/nuki/config_flow.py

```diff
@@ -59,9 +59,9 @@ def __init__(self):
         self.discovery_schema = {}
         self._data = {}
 
-    def parse_id(self, id):
+    def parse_id(hardware_id):
```

Comment:
Move this to be a standalone helper function that we can use both here and in the config entry setup id migration.